pySDC/projects/Monodomain/datatype_classes/my_mesh.py

```diff
@@ -0,0 +1,9 @@
+from pySDC.implementations.datatype_classes.mesh import MultiComponentMesh
+
+
+class imexexp_mesh(MultiComponentMesh):
+    components = ['impl', 'expl', 'exp']
+
+
+def get_np_list(vec):
```

Comment:
What is the purpose of this function? I assume `vec` should be of time `imexexp_mesh`, so so you can directly interface the components from that instead of splitting it apart into a list first.

Reply:
yes, it was a temporary trick for compatibility with the c++ interface. Now it's fixed.